elfloader-tool/CMakeLists.txt

```diff
@@ -281,7 +281,6 @@ if(DEFINED platform_yaml)
         VERBATIM
         DEPENDS ${platform_yaml} ${PLATFORM_SIFT}
     )
-    set_property(SOURCE src/common.c PROPERTY OBJECT_DEPENDS ${PLATFORM_INFO_H})
```

Comment:
I don't know how CMake can find this dependency automatically.  It seems like CMake is doing too much magic here.  But according to the docs this does seem able to be removed. 
"This property need not be used to specify the dependency of a source file on a generated header file that it includes. Although the property was originally introduced for this purpose, it is no longer necessary. If the generated header file is created by a custom command in the same target as the source file, the automatic dependency scanning process will recognize the dependency"

Reply:
CMake can't find it automatically. When the compiler has run the first time, dep files are created.This is where ninja gets them from and know how to rebuils thing. It just gets tricky when there is no ninja rule that created them. Then CMake has to create them when the build workspace is initialized. But that create would be hard-coded then anyway in the CMake scripts. So the smarter solution in my opinion is always, have a rule that can create things on demand by ninja. Avoid leaving this in CMake, so CMake does not even have to be invoked at all for a re-build.